lib/redis_memo/thread_local_var.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module RedisMemo::ThreadLocalVar
+  def self.define(var_name) # :nodoc:
+    thread_key = :"__redis_memo_#{var_name}__".freeze
+    const_set(var_name.to_s.upcase, thread_key)
```

Comment:
Are these constants being used anywhere? Most call sites are using the method ThreadLocalVar.{var_name}?

Reply:
Ah, I was using this to make sure var_names are uniquely defined globally.

If we define two ThreadLocalVars with the same var_name, there should be an error.

I will add a test case covering this behavior!